packages/server/database/migrations/20220112175838-addMissingPlainTextContentToTask.ts

```diff
@@ -0,0 +1,46 @@
+import {R} from 'rethinkdb-ts'
+import extractTextFromDraftString from 'parabol-client/utils/draftjs/extractTextFromDraftString'
+
+export const up = async function (r: R) {
+  // We have about 2500 such tasks
+  const missedTasks = await r
+    .table('Task')
+    .filter((row) => row.hasFields('plaintextContent').not())
```

Comment:
+1 i'm a little concerned about the 2500 rows, i know we've crashed when doing batches of <10,000 but I can't remember exactly what the size was. As long as we deploy during an offpeak time I'm willing to roll the dice.

For future reference, skip/limit and using `,pluck('id', 'content')` are great at keeping things well under memory constraints.

Reply:
@mattkrick I didn't split into batches as I saw a similar migration that uses batch size = 10000: https://github.com/ParabolInc/parabol/blob/5e04ddd020f21692c9c6547dede7fc73064fb54f/packages/server/database/migrations/20201008134420-addPlainTextContentToTask.ts#L36

So I thought that 2500 would be fine too.

I can split on smaller batches if needed.